CONTRIBUTING.rst

```diff
@@ -4,29 +4,37 @@ Instruction for contributors
 Developer environment
 ---------------------
 
-First clone the git repo::
+First clone the git repo
 
-   $ git clone git@github.com:aio-libs/aiopg.git
-   $ cd aiopg
+.. code-block:: shell
+
+    $ git clone git@github.com:aio-libs/aiopg.git
+    $ cd aiopg
 
 After that you need to create and activate a virtual environment.  I
 recommend using :term:`virtualenvwrapper` but just :term:`virtualenv` or
-:term:`venv` will also work.  For ``virtualenvwrapper``::
+:term:`venv` will also work. For :term:`virtualenvwrapper`
+
+.. code-block:: shell
+
+    $ mkvirtualenv aiopg -p `which python3`
 
-   $ mkvirtualenv aiopg -p `which python3`
+For `venv` (for example; put the directory wherever you want)
 
-For ``venv`` (for example; put the directory wherever you want)::
+.. code-block:: shell
 
-   $ python3 -m venv ../venv_directory
-   $ source ../venv_directory/bin/activate
+    $ python3 -m venv ../venv_directory
+    $ source ../venv_directory/bin/activate
 
-Just as when doing a normal install, you need the :term:`libpq` library::
+Just as when doing a normal install, you need the :term:`libpq` library
 
-   $ sudo apt-get install libpq-dev
+.. code-block:: shell
+
+    $ sudo apt-get install libpq-dev
 
 **UPD**
 
-The latest ``aiopg`` test suite uses docker container for running
+The latest `aiopg` test suite uses docker container for running
```

Comment:
In RST single backticks produce `italic text` while double backticks make ``inline code``.

```suggestion
The latest ``aiopg`` test suite uses docker container for running
```